README.md

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
This is a side effect of some unfinished merge conflict. In this project It's Ok to leave as is. In general it should be resolved.